src/main/java/org/galatea/starter/Application.java

```diff
@@ -25,7 +26,10 @@
   public static void main(final String[] args) {
     log.info("Starting spring application {}", System.getProperty("application.name"));
     SpringApplication.run(Application.class, args);
-
+    String a = args[1];
```

Comment:
Are we guaranteed that this will always be args[1]?
Can you rename the variable to something more descriptive?